backoffice/backoffice/workflows/api/views.py

```diff
@@ -169,9 +170,14 @@ def resolve(self, request, pk=None):
             )
             utils.add_decision(pk, request.user, serializer.validated_data["value"])
 
-            return airflow_utils.trigger_airflow_dag(
+            airflow_utils.trigger_airflow_dag(
                 ResolutionDags[serializer.validated_data["value"]].label, pk, extra_data
             )
+            workflow_serializer = self.serializer_class(
+                get_object_or_404(Workflow, pk=pk)
```

Comment:
are you sure we need to do a query here? The object is already changed in the session